content/en/cloud_cost_management/anomalies.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Anomalies Page
+further_reading:
+- link: "/cloud_cost_management/"
+  tag: "Documentation"
+  text: "Cloud Cost Management"
+- link: "/cloud_cost_management/monitors"
+  tag: "Monitors"
+  text: "Create Cost Monitors"
+---
+
```

Comment:
Seems like this feature might be in preview, does this page need a preview banner? Happy to help you add one!

Reply:
it will be GA next week, so we'll just avoid pushing the docs until it's GA!!